Remove vendors footer and add SFDS link #169017078

james-wills-sf opened the pull request:
[Pivotal #169017078](https://www.pivotaltracker.com/n/projects/1405352)

## Summary
Remove the app footer that says "made with <3 from Exigy and Vertiba"
and update the "in partnership with" list to include SFDS and link to
the SFDS site.

Additonally:

- Remove "del" from spanish translation. It's not necessary
- Swap the link text for Spanish, the links were going to the wrong locations
- Update the translations for Spanish, Chinese, Tagalog to say "San Francisco Digital Services"

## Screenshot (before)
<img width="1401" alt="Screen Shot 2020-04-23 at 3 40 01 PM" src="https://user-images.githubusercontent.com/64036574/80156791-a2f93700-8579-11ea-9311-9d8a8e6d1181.png">


## Screenshot (after)
Note that there is no footer at the bottom anymore that calls out Vertiba and Exigy, and `San Francisco Digital Services` has replaced the `San Francisco Department of Technology` link.

<img width="1396" alt="Screen Shot 2020-04-23 at 3 27 49 PM" src="https://user-images.githubusercontent.com/64036574/80156883-d20fa880-8579-11ea-993d-944e8a321e10.png">



## Screenshot (after clicking on "San Francisco Digital Services")
<img width="1402" alt="Screen Shot 2020-04-23 at 3 27 35 PM" src="https://user-images.githubusercontent.com/64036574/80156762-8f4dd080-8579-11ea-86c0-802e156723a6.png">


## Comparison: English old / new
<img width="1154" alt="Screen Shot 2020-04-28 at 1 58 30 PM" src="https://user-images.githubusercontent.com/64036574/80543828-9bb09f80-8964-11ea-957e-a9f57cd72695.png">

## Comparison: Spanish old / new
<img width="1287" alt="Screen Shot 2020-04-28 at 1 58 04 PM" src="https://user-images.githubusercontent.com/64036574/80543824-96ebeb80-8964-11ea-9fec-54ef05fe8820.png">

## Comparison: Chinese old / new
<img width="1374" alt="Screen Shot 2020-04-28 at 3 24 45 PM" src="https://user-images.githubusercontent.com/64036574/80543788-86d40c00-8964-11ea-830d-adcbe83f8554.png">

## Comparison: Tagalog old / new
<img width="1500" alt="Screen Shot 2020-04-28 at 1 57 48 PM" src="https://user-images.githubusercontent.com/64036574/80543810-90f60a80-8964-11ea-939b-62610fc91799.png">

## After updating office of innovation translations
![Screen Shot 2020-04-29 at 3 59 20 PM](https://user-images.githubusercontent.com/64036574/80655079-a6396a80-8a32-11ea-8bbb-74636da82373.png)
![Screen Shot 2020-04-29 at 3 59 29 PM](https://user-images.githubusercontent.com/64036574/80655082-a8032e00-8a32-11ea-834c-14409ad224be.png)
![Screen Shot 2020-04-29 at 3 59 37 PM](https://user-images.githubusercontent.com/64036574/80655085-a89bc480-8a32-11ea-861b-866c172e4570.png)

akegan commented:
Not really in the scope of the ticket but I just noticed that the mayor's office of innovation link is a dead link. Seems like it should be https://www.innovation.sfgov.org/ if you're up for updating that too!

akegan commented on app/assets/json/translations/locale-es.json:
```diff
@@ -442,7 +442,7 @@
             "city_county_of_sf": "Ciudad y condado de San Francisco",
             "contact": "Contacto",
-            "dahlia_description": "<p class='c-iron'>DAHLIA: El Portal de Vivienda de San Francisco es un proyecto de la <br/><a class='a-white' href='{{mohcdUrl}}' target='_blank'>Oficina de Desarrollo Comunitario y de Viviendas del Alcalde</a></p><p class='t-small c-iron margin-top'>en colaboración con la<br/><a class='a-white' href='{{sfDeptUrl}}' target='_blank'>Oficina de Innovación Cívica d ...
+            "dahlia_description": "<p class='c-iron'>DAHLIA: El Portal de Vivienda de San Francisco es un proyecto de la <br/><a class='a-white' href='{{mohcdUrl}}' target='_blank'>Oficina de Desarrollo Comunitario y de Viviendas del Alcalde</a></p><p class='t-small c-iron margin-top'>en colaboración con la<br/><a class='a-white' href='{{sfdsUrl}}' target='_blank'>Oficina de Innovación Cívica del ...
```
this might be a case where we could delete the translation because it's no longer accurate 🤔. Not a big deal, just a thought. 

We might be able to also update the translations without having to go to the vendor if we know how to translate "San Francisco Digital Services" in all of our supported languages.

james-wills-sf replied:
@akegan if we delete the translations, will it just use the English name until the new translations are added?

akegan replied:
It should just google translate the whole string. I asked in #topic-translations yesterday and it doesn't sound like we have translations for San Francisco Digital Services unfortunately so I think deleting it might be the best bet for now.

james-wills-sf commented:
@akegan I made both of the changes you requested, let me know if you want to take another look! Otherwise I'll just merge it as-is

akegan commented:
ha great timing, we just got the translations back 😬 

https://sfdigitalservices.slack.com/archives/CKJL9TRU2/p1588103497096900?thread_ts=1586484530.028200&cid=CKJL9TRU2

james-wills-sf commented:
@akegan I think this is ready for re-review! Note that I didn't just add new translations for the SFDS text, I also found a bug with spanish where the link locations were flipped so I flipped them back, and I removed an unnecessary "del" from the spanish text and changed the Chinese text surrounding the SFDS label to what Sharon says is best.

[Slack discussion about Spanish and Chinese specifics](https://sfdigitalservices.slack.com/archives/CKJL9TRU2/p1588104371100200)

I'm still waiting on confirmation about the chinese translation, but the code shouldn't change significantly.
